CONTRIBUTING.md

```diff
@@ -73,8 +73,8 @@ npm run storybook
     Demos all features | All interactions spec'd out in the Design Guide has visual support and a demo as proof.
     Documented/annotated | All rule blocks, placeholders, mixins, and functions has associated comment blocks that convey purpose and list inline notations referencing declarations that need further explanation.
     Matches UI/UX specs | All features are pixel perfect when compared to the Design Guide.
-    Meets the [code style guide](https://github.com/IBM/carbon-charts/wiki/Code-style-guide) | All code adheres to our custom style guide giving off the impression that it was written by one developer.
-    Accessible | All code passes the [Dynamic Assessment Plugin (DAP)](https://w3-connections.ibm.com/wikis/home?lang=en-us#!/wiki/W88ee03f8907c_412b_a3a8_988dabb72b35/page/Dynamic%20Assessment%20Plugin) scan in Chrome.
+    All new code follows similar structure and style to the existing codebase giving off the impression that it was written by one developer.
```

Comment:
Although `give off` could be an accurate representation of what following a similar structure and style accomplishes, it could also be [easily misunderstood](https://www.urbandictionary.com/define.php?term=give%20off) ;)